salt/metalk8s/saltmaster/files/salt-master-pod.yaml.j2

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: saltmaster
+  namespace: kube-system
+  labels:
+    app: saltmaster
+    app.kubernetes.io/name: saltmaster
+    app.kubernetes.io/version: '{{ saltmaster_version }}'
+    app.kubernetes.io/component: salt
+    heritage: metalk8s
+    app.kubernertes.io/part-of: metalk8s
+    app.kubernetes.io/managed-by: salt
+spec:
+  hostNetwork: true
+  priorityClassName: system-cluster-critical
+  securityContext:
+    runAsUser: 0
+    fsGroup: 0
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+  containers:
+    - name: saltmaster
+      image: docker.io/library/{{saltmaster_image}}:{{saltmaster_version}}
+      imagePullPolicy: Never
+      ports:
+        - name: master
+          containerPort: 4505
+          protocol: TCP
+        - name: api
+          containerPort: 4506
+          protocol: TCP
+      volumeMounts:
+        - name: config
+          mountPath: '/etc/salt'
+        - name: cache
+          mountPath: '/var/cache/salt'
+        - name: run
+          mountPath: '/var/run/salt'
+        - name: log
```

Comment:
No, we can't have this. Instead, Salt should only log to `stdout`, and not use a logfile.

Reply:
removed